src/xunit.performance.analysis/Program.cs

```diff
@@ -41,7 +45,7 @@ private static int Main(string[] args)
 
             for (int i = 0; i < args.Length; i++)
             {
-                if (args[i].StartsWith("-") || args[i].StartsWith("/"))
```

Comment:
The Linux file paths start with /home and it ended up reading those as parameters, I could probably add a #if for Linux here but thought this would be more consistent